### ISSUE TITLE
How keyboard navigation of persons list works if user does not have arrow keys

### ISSUE THREAD

**Issue by Eugene-Ong-W-X** [Eugene-Ong-W-X/ped#11]
![image.png](https://raw.githubusercontent.com/Eugene-Ong-W-X/ped/main/files/20c6f72c-e063-43d8-9c51-d173ece83c76.png)

Could it be possible to provide an alternative keyboard-friendly way to navigate for smaller keyboards that do not provide arrow keys?
